trailblazer/services/tower/tower_api_service.py

```diff
@@ -31,7 +31,9 @@ def cancel_jobs(self, analysis_id: int) -> None:
     def get_status(self, analysis_id: int) -> TrailblazerStatus:
         analysis: Analysis = self.store.get_analysis_with_id(analysis_id)
         response = self.client.get_workflow(analysis.tower_workflow_id)
-        status = TOWER_WORKFLOW_STATUS.get(response.workflow.status, TrailblazerStatus.ERROR)
+        status: TrailblazerStatus = TOWER_WORKFLOW_STATUS.get(
+            response.workflow.status, TrailblazerStatus.ERROR
+        )
```

Comment:
Can you revert this and check you linting settings? This line is only 94 characters and we use 100 for line breaks

Reply:
<img width="905" alt="Screenshot 2025-01-16 at 13 27 54" src="https://github.com/user-attachments/assets/a67bfc62-58dd-4b46-8e1c-db6db0199c59" />


I think it's longer than 100? :)